src/stream-ciphers.rst

```diff
@@ -433,49 +433,48 @@ Bob decrypts :math:`C^{\prime}`, and gets the three plaintext blocks
                 & = P_1 \xor IV
    \end{aligned}
 
-:math:`R` is some random block. Its value doesn't matter.
+:math:`R` is a random block. Its value does not matter.
 
-Under the chosen-ciphertext attack assumption, Mallory recovers that
-decryption. She is only interested in the first block
+Mallory recovers the decryption under the chosen-ciphertext attack assumption.
+She is only interested in the first block
 (:math:`P^{\prime}_1 =
 P_1`) and the third block (:math:`P^{\prime}_3 = P_1 \xor IV`). By
-XORing those two together, she finds
-:math:`(P_1 \xor IV) \xor P_1 = IV`. But, the IV is the key, so Mallory
-successfully recovered the key by modifying a single message.
-
-Lesson learned: don't use the key as an IV. Part of the fallacy in the
-introduction is that it assumed secret data could be used for the IV,
-because it only had to be unpredictable. That's not true: “secret” is
-just a different requirement from “not secret”, not necessarily a
-*stronger* one. It is not generally okay to use secret information where
-it isn't required, precisely because if it's not supposed to be secret,
+XORing the two together, she finds
+:math:`(P_1 \xor IV) \xor P_1 = IV`. IV is the key, so Mallory
+successfully recovers the key by modifying a single message.
+
+Lesson learned: do not use the key as an IV. Part of the fallacy in the
+introduction is assuming secret data can be in the IV given
+it only has to be unpredictable. This is not true: “secret” is
+just a different requirement from “not secret”. It does not mean a
+*stronger* one. Secret information should not be used when
+not required. Precisely because if the secret is supposed to be secret,
 the algorithm may very well treat it as non-secret, as is the case here.
-There *are* plenty of systems where it is okay to use a secret where it
-isn't required. In some cases you might even get a stronger system as a
-result, but the point is that it is not generally true, and depends on
-what you're doing.
+Plenty of systems exist where it is okay to use a secret when not
+required. In some cases you may get a stronger system.
+Though it depends on what you are doing.
```

Comment:
Again this new paragraph is overall worse than the original IMO

Reply:
Reverted